Add missing delegated_auth_accessors config field to /sys/mounts/<path> response (#26876)
hashicorp/vault: website/content/docs/commands/secrets/enable.mdx

- `-allowed-managed-keys` `(string: "")` - Managed key name(s) that the mount
  in question is allowed to access. Note that multiple keys may be specified
  by providing this option multiple times, each time with 1 key.

> - `-delegated-auth-accessors` `(string: "")` - A list of permitted authentication
>   accessors this backend can delegate authentication to. Note that multiple accessors
>   may be specified by providing this option multiple times, each time with 1 accessor.

Reviewer: ```suggestion
- `-delegated-auth-accessors` `(string: "")` - An authorized accessors the auth
  backend can delegate authentication to. To allow multiple accessors, provide
  the `delegated-auth-accessors` multiple times, each time with 1 accessor.
```
Style correction: write in active voice, avoid "this" as a pronoun.

If folks have to provide the parameter multiple times to set multiple accessors, technically the parameter is not a list (even if we create a list in the backend), no?

Author: > If folks have to provide the parameter multiple times to set multiple accessors, technically the parameter is not a list (even if we create a list in the backend), no?

Yup you are correct, accepted your updates but tweaked to "An authorized accessor" for both. Thanks for the feedback!
